projects/azure/azure-cicd/secret-refresh-job.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: token-refresh
+  namespace: azure
+spec:
+  schedule: "0 0 1,15 * * "  
```

Comment:
(I know this has merged but)
@kargakis do we/can we get notifications if this fails?
@mjudeikis better to do this daily, otherwise a vm can be built which only has a token lifetime of 15 days?

Reply:
Agreed about daily refreshes. Re notifications, we can get emails if we move this to run as a prow periodic job and https://github.com/kubernetes/test-infra/issues/8668 is addressed.